java/client/src/org/openqa/selenium/support/ui/FluentWait.java

```diff
@@ -140,12 +161,30 @@ public FluentWait(T input, Clock clock, Sleeper sleeper) {
    * In reality, the interval may be greater as the cost of actually evaluating a condition function
    * is not factored in. The default polling interval is {@link #FIVE_HUNDRED_MILLIS}.
    *
+   * @deprecated use {@link #pollingEvery(long, TemporalUnit)}
+   *
    * @param duration The timeout duration.
    * @param unit The unit of time.
    * @return A self reference.
    */
+  @Deprecated
   public FluentWait<T> pollingEvery(long duration, TimeUnit unit) {
-    this.interval = new Duration(duration, unit);
+    return pollingEvery(duration, toChronoUnit(unit));
+  }
+
+  /**
+   * Sets how often the condition should be evaluated.
+   *
+   * <p>
+   * In reality, the interval may be greater as the cost of actually evaluating a condition function
+   * is not factored in. The default polling interval is {@link #DEFAULT_WAIT_DURATION}.
+   *
+   * @param duration The timeout duration.
+   * @param unit The unit of time.
+   * @return A self reference.
+   */
+  public FluentWait<T> pollingEvery(long duration, TemporalUnit unit) {
```

Comment:
Again, I'm not sure this is adding much to the user-facing APIs. `TimeUnit` is part of the concurrency package, and so ships with Java.

Reply:
Same reasoning as before.